swagger_to_sdk_config_track2.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/sdkautomation/SwaggerToSdkConfigSchema.json",
+  "advancedOptions": {
+    "createSdkPullRequests": true,
+    "generationCallMode": "one-for-all-configs"
```

Comment:
Is this required? I thought this was the default.

Reply:
That goes for all the advancedOptions. I think we can remove them and only have the init and generate script configurations.

Reply:
I'm not sure about it. @PhoenixHe-msft will update the configs after we merge this PR.